resources/testdriver.js

```diff
@@ -693,68 +693,44 @@
          *
          * @param {String} sensor_type - The `MockSensorType
          * <https://w3c.github.io/sensors/#enumdef-mocksensortype>`_ identifier.
-         * @param {Object} sensor_params - An object for sensor parameters.
-         * "``connected``" key holds boolean value. "``connected``" indicates a
-         * mock sensor's connection flag which is used for switching the
-         * connection between Sensor object and mock sensor. When set to false
-         * the user agent must force the result of invoking connect to sensor
-         * with mock sensor's associated Sensor object as argument to false,
-         * otherwise true.
-         * "``minSamplingFrequency``" key holds a double representing frequency
-         * in Hz that is used to set minimum supported sampling frequency for
-         * the associated mock sensor.
-         * "``maxSamplingFrequency``" key holds a double representing frequency
-         * in Hz that is used to set maximum supported sampling frequency for
-         * the associated mock sensor.
+         * @param {MockSensor} sensor_params - A `MockSensor
```

Comment:
`MockSensor` is something else, `sensor_params` is a `MockSensorConfiguration` dictionary.